src/ui/screens/settings/ViewTipsAndTricks.qml

```diff
@@ -0,0 +1,192 @@
+                        onClicked: featureTourPopup.startTour();
+                    }
+
+                    Repeater {
+                        model: tipsSections.count
+
+                        Column {
+                            property var section: tipsSections.get(index)
+
+                            visible: guidesList.count || tutorialsList.count
+
+                            Layout.fillWidth: true
+                            Layout.alignment: Qt.AlignTop
+                            Layout.topMargin: VPNTheme.theme.vSpacingSmall
+                            spacing: VPNTheme.theme.vSpacing
+
+                            VPNGuideList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: guidesList
+                                title: section.title
+                                description: section.description
+                                customGuideFilter: section.filter
+                            }
+
+                            VPNTutorialList {
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+
+                                id: tutorialsList
+                                customTutorialFilter: section.filter
+                            }
+                        }
+                    }
+                }
+            },
+
+            // Tutorials
+            VPNFlickable {
+                flickContentHeight: layoutTutorial.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutTutorial
+
+                    anchors.fill: parent
+                    anchors.topMargin: VPNTheme.theme.vSpacing
+                    anchors.bottomMargin: VPNTheme.theme.vSpacing
+                    anchors.leftMargin: VPNTheme.theme.windowMargin
+                    anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                    VPNTutorialList { 
+                        Layout.fillWidth: true
+                    }
+                }
+            },
+
+            // Tips
+            VPNFlickable {
+                flickContentHeight: layoutGuide.implicitHeight + (VPNTheme.theme.vSpacing * 2)
+                interactive: flickContentHeight > height
+
+                ColumnLayout {
+                    id: layoutGuide
+
+                    anchors.fill: parent
+                    anchors.topMargin: VPNTheme.theme.vSpacing
+                    anchors.bottomMargin: VPNTheme.theme.vSpacing
+                    anchors.leftMargin: VPNTheme.theme.windowMargin
+                    anchors.rightMargin: VPNTheme.theme.windowMargin
+
+                    Repeater {
+                        model: tipsSections.count
+
+                        Column {
```

Comment:
So I needed this `Column` here because without it:
1) I can't access the `tipsSections` data, although I got no error... Just no data. Go figure... but apparently you can't access ObjectModel data inside a Repeater directly, gotta have an inner element. 
2) I tried using what I thought would be a meaningless `Item` as a wrapper, but then suddenly QML doesn't know how to compute heights. I don't like this, we now have ColumnLayout > Column > ColumnLayout, why are so many columns necessary for it to compute heights and widths correctly? Is there a better way? :sos:

Reply:
You shouldn't need the column - just put `property var section: tipsSections.get(index)` inside the delegate (VPNGuideList)

Reply:
...I should have seen that one. Thanks!